### PR TITLE
Add TLS params to memcached

### PR DESCRIPTION
**Pull request by moisesguimaraes**
These params are required for Memcached TLS testing.

### CODE REVIEW

File: pifpaf/drivers/memcached.py [hunk 1]
```diff
@@ -30,14 +34,37 @@ def get_options(cls):
...
-                           "-p " + str(self.port)],
-                          wait_for_port=self.port)
+        command = ["memcached", "-p " + str(self.port)]
+
+        if self.ssl_chain_cert:
+            command.append("-Z")
+            command.append("-o")
+            command.append("ssl_verify_mode=1")
+            command.append("-o")
+            command.append("ssl_chain_cert=" + self.ssl_chain_cert)
```
Comment (jd) [inline pifpaf/drivers/memcached.py:55]:
```suggestion
            command.extend(["-Z", "-o" "ssl_verify_mode=1", "-o", "ssl_chain_cert=" + self.ssl_chain_cert])
```

File: pifpaf/drivers/memcached.py [hunk 2]
```diff
@@ -30,14 +34,37 @@ def get_options(cls):
...
+
+            if self.ssl_key:
+                command.append("-o")
+                command.append("ssl_key=" + self.ssl_key)
```
Comment (jd) [inline pifpaf/drivers/memcached.py:59]:
```suggestion
                command.extend(["-o", "ssl_key=" + self.ssl_key])
```

File: pifpaf/drivers/memcached.py [hunk 3]
```diff
@@ -30,14 +34,37 @@ def get_options(cls):
...
+
+            if self.ssl_ca_cert:
+                command.append("-o")
+                command.append("ssl_ca_cert=" + self.ssl_ca_cert)
```
Comment (jd) [inline pifpaf/drivers/memcached.py:63]:
```suggestion
                command.extend(["-o", "ssl_ca_cert=" + self.ssl_ca_cert])
```

File: pifpaf/drivers/memcached.py [hunk 4]
```diff
@@ -30,14 +34,37 @@ def get_options(cls):
...
+
+        print(command)
```
Comment (jd) [inline pifpaf/drivers/memcached.py:65]:
```suggestion
```

File: pifpaf/drivers/memcached.py [hunk 5]
```diff
@@ -30,14 +34,37 @@ def get_options(cls):
...
+
```
Comment (jd) [inline pifpaf/drivers/memcached.py:66]:
```suggestion
```

File: .gitignore [hunk 6]
```diff
@@ -1,5 +1,6 @@
 build
 *.egg-info
+*.eggs
```
Comment (jd) [inline .gitignore:3]:
```suggestion
```
unrelated

**Comment by jd** [comment]
I don't know why Travis does not run, but it fails anyhow.

**Comment by jd** [comment]
CI is broken but tests to be ok so let's merge manually

**Comment by moisesguimaraes** [comment]
CI is probably failing due to openstack dependencies now requiring python 3.6+

**Comment by moisesguimaraes** [comment]
Do you have a timeline for this change to reach pypi?
